src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -534,7 +540,7 @@ private void AddTupleTypeName(INamedTypeSymbol symbol)
                 if (element.IsExplicitlyNamedTupleElement)
                 {
                     AddSpace();
-                    builder.Add(CreatePart(SymbolDisplayPartKind.FieldName, symbol, element.Name));
+                    builder.Add(CreatePart(SymbolDisplayPartKind.FieldName, element, element.Name));
```

Comment:
this was a bug before (we were adding in the tuple again for each tuple element).  This caused problems with the IDE code as we'd then try to expand these references out as well.